src/core/server/plugins/plugins_system.ts

```diff
@@ -36,7 +37,7 @@ export class PluginsSystem {
     this.plugins.set(plugin.name, plugin);
   }
 
-  public async startPlugins() {
+  public async startPlugins(deps: PluginsServiceStartDeps) {
```

Comment:
__note:__ if we ever want to limit `PluginService` start deps that `PluginSystem` can have access to, we can easily do that, but right now I don't see any problem with sharing entire set of deps.